test/Reflection/typeref_decoding.swift

```diff
@@ -10,6 +10,7 @@
 
 // FIXME: rdar://127796117
 // UNSUPPORTED: OS=linux-gnu && CPU=aarch64
+// XFAIL: OS=linux-android
```

Comment:
Do you have the failure handy? I don't know if I'm allowed to copy the failure from the radar, but I can confirm if it's the same as the issue making linux-gnu aarch64 unsupported. If they are the same issue, it's probably worth keeping everything on the same line so that they get cleaned up together and I can make a note in the radar.

Reply:
@adrian-prantl says both are [passing on linux AArch64](https://github.com/swiftlang/swift/pull/82325#issuecomment-3002136570) now. 🤷